client/src/App.js

```diff
@@ -1,87 +1,91 @@
-import React, { Component } from 'react';
-import axios from 'axios'
-import { Route } from 'react-router-dom'
+import React, { Component } from "react";
+import axios from "axios";
+import { Route } from "react-router-dom";
 // components
-import Signup from './components/sign-up'
-import LoginForm from './components/login-form'
-import Navbar from './components/Nav/index'
-import Home from './components/Home/home'
-import CreateJob from './components/CreateJob/createjob'
-import AddChild from './components/AddChild/addchild';
+import Signup from "./components/sign-up";
+import LoginForm from "./components/login-form";
+import Navbar from "./components/Nav/index";
+import Home from "./components/Home/home";
+import CreateJob from "./components/CreateJob/createjob";
+import About from "./components/About/about";
+import AddChild from "./components/AddChild/addchild";
+import Shop from "./components/Shop/shop";
+import Button from "react-bootstrap/Button";
+import MyVerticallyCenteredModal from "./components/Modals/index";
 
 class App extends Component {
-  constructor() {
-    super()
+  constructor(...args) {
+    super(...args);
     this.state = {
       loggedIn: false,
-      username: null
-    }
+      modalShow: false
+    };
 
-    this.getUser = this.getUser.bind(this)
-    this.componentDidMount = this.componentDidMount.bind(this)
-    this.updateUser = this.updateUser.bind(this)
+    this.getUser = this.getUser.bind(this);
+    this.componentDidMount = this.componentDidMount.bind(this);
+    this.updateUser = this.updateUser.bind(this);
   }
 
   componentDidMount() {
-    this.getUser()
+    this.getUser();
   }
 
-  updateUser (userObject) {
-    this.setState(userObject)
+  updateUser(userObject) {
+    this.setState(userObject);
   }
 
   getUser() {
-    axios.get('/user').then(response => {
-      console.log('Get user response: ')
-      console.log(response.data)
+    axios.get("/user").then(response => {
+      console.log("Get user response: ");
```

Comment:
debugging console.log. I dont think we need this in master.

Reply:
Removed console logs, many console logs.